content/en/docs/ops/configuration/traffic-management/tls-configuration/index.md

```diff
@@ -119,8 +119,8 @@ For TLS connections, there are a few more options:
     For passthrough traffic, configure the TLS mode field to `PASSTHROUGH`:
 
     {{< text yaml >}}
-    apiVersion: networking.istio.io/v1beta1
-    kind: Gateway
+    apiVersion: networking.istio.io/v1
+.   kind: Gateway
```

Comment:
Looks like there's multiple of these

Reply:
That's what I get from taking the suggestion from prow